lib/active_remote/base.rb

```diff
@@ -39,6 +40,10 @@ class Base
     # so it needs to be included last.
     include Dirty
 
+    # Overrides persistence methods, so it must included after
+    include Validations
```

Comment:
After looking at the `Validations` module, this shouldn't be an issue since it calls `super`. However, if there are not specs around this already, it might a good idea to add some to verify.